egs/snips/v1/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,340 @@
+#!/bin/bash
+#
+# Copyright 2019  Johns Hopkins University (Author: Daniel Povey)
+#           2019  Yiming Wang
+# Apache 2.0
+
+
+set -e
+
+# configs for 'chain'
+stage=0
+nj=30
+gmm=mono
+train_stage=-5 # starting from -5 to skip phone-lm estimation
+get_egs_stage=-10
+affix=1a
+remove_egs=false
+xent_regularize=0.1
+
+# training options
+srand=0
+num_epochs=3
+num_jobs_initial=2
+num_jobs_final=5
+chunk_width=140,100,160
+common_egs_dir=
+reporting_email=
+dim=80
```

Comment:
I think I'd prefer to have dim and bn_dim defined in the xconfig section if you're done tuning, just to make them easier to read.  OK if you need them here for some reason.